presto-main/src/main/resources/webapp/src/components/LivePlan.jsx

```diff
@@ -125,10 +125,19 @@ class PlanNode extends React.Component<PlanNodeProps, PlanNodeState> {
     }
 
     render() {
+        const details = this.props.details;
+        var distribution = "";
+
+        if (details.includes("Distribution")) {
+            var index = details.indexOf("Distribution");
+            var distribution = details.slice(index, details.length).split("\n")[0].split(" ")[1];
+            distribution = " (" + distribution.slice(0,1) + distribution.slice(1, distribution.length).toLowerCase() + ")";
```

Comment:
We should preserve the value coming from the server and not change capitalization.